### PR TITLE
Fixes slime processor giving an extra core

### LINKED ISSUE #13156: Slime Processor: Doubled Output
At the start of the round, the slime processor in xenobio will produce 2 core for every slime put into it instead of just a singular 1, like its meant to.

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Fixes #13156

### CODE REVIEW

**Comment by duncathan** [comment]
Nice

**Comment by LatD** [comment]
I think you changed it to the wrong direction. Now it gives 3 cores per slime. (For loop going 0, 1, 2)
C + processor.rating is 2.

Also since this has become a feature to speed up the boring and slow xenobio, the slime core variable might need to be changed to 2 if enough people want this

**Comment by Bawhoppen** [comment]
may have been not intended originally, but I agree its for the best to leave it in

File: code/modules/food&drinks/kitchen machinery/processor.dm [hunk 1]
```diff
@@ -86,7 +86,7 @@
 		S.loc = loc
 		S.visible_message("<span class='notice'>[C] crawls free of the processor!</span>")
 		return
-	for(var/i = 1, i <= C + processor.rating_amount, i++)
+	for(var/i = 1, i < C + processor.rating_amount, i++)
```
Comment (RemieRichards) [inline code/modules/food&drinks/kitchen machinery/processor.dm]: bad for loop! BAD FOR LOOP.

if you're doing ```< C + processor.rating_amount```
it's:

```for(var/i in 1 to (C+processor.rating_amount-1))```

Comment (KorPhaeron) [inline code/modules/food&drinks/kitchen machinery/processor.dm]: This is goofs loop dont look at me I'd have done i++ inside the loop

Comment (RemieRichards) [inline code/modules/food&drinks/kitchen machinery/processor.dm]: FIX THE LOOOOP.

Comment (KorPhaeron) [inline code/modules/food&drinks/kitchen machinery/processor.dm]: Fiiiine

**Comment by KorPhaeron** [comment]
@tkdrg 

Took me 3 days but I did the one line change

**Pull request merged**